packages/core/src/domain/tracekit/computeStackTrace.spec.ts

```diff
@@ -499,6 +499,12 @@ Error: foo
     })
   })
 
+  it('should ignore custom error messages ', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you clarify test description?